### ISSUE TITLE
cmp not defined. Basically doesn't work with python 3?

### ISSUE THREAD

**Issue by jakern** [joshvillbrandt/wireless#32]
```python
>>> w = Wireless()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\site-packages\wireless\Wireless.py", line 23, in __init__
    self._driver_name = self._detectDriver()
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\site-packages\wireless\Wireless.py", line 50, in _detectDriver
    compare = self.vercmp(ver, "0.9.9.0")
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\site-packages\wireless\Wireless.py", line 71, in vercmp
    return cmp(normalize(actual), normalize(test))
NameError: name 'cmp' is not defined
```

**Comment by AIIX** [comment]
Same error with Python 3.6 and pip install wireless: 
return cmp(normalize(actual), normalize(test))
NameError: name 'cmp' is not defined

The pypi version is not upto date with the git version, request to please upload the latest fixes to pypi release

**Comment by joshvillbrandt** [comment]
Released as 0.3.3 on pypi.

https://pypi.org/project/wireless/
